content/docs/ui/sending-email/email-testing.md

```diff
@@ -11,14 +11,7 @@ seo:
   override: true
   Description: How to use SendGrid's new email testing feature to see how your email shows up in different inboxes and across different devices and operating systems.
```

Comment:
How to use SendGrid's new email testing feature to preview your email renders, performs against spam filters, and validate the links within your email.